master/internal/experiment/bulk_action_intg_test.go

```diff
@@ -30,30 +30,64 @@ func TestGetExperimentsEditableByUser(t *testing.T) {
 	testUser := db.RequireMockUser(t, db.SingleDB())
 	testWorkspaceID, _ := db.RequireMockWorkspaceID(t, db.SingleDB(), "TestGetExperimentsEditableByUser-"+nameExt.String())
 	testProjectID, _ := db.RequireMockProjectID(t, db.SingleDB(), testWorkspaceID, false)
+	testProjectID2, _ := db.RequireMockProjectID(t, db.SingleDB(), testWorkspaceID, false)
 
-	testModelStates := []model.State{
-		model.ActiveState,
-		model.CanceledState,
-		model.CompletedState,
-		model.ErrorState,
-		model.PausedState,
-		model.StoppingCanceledState,
-		model.DeletingState,
-		model.RunningState,
+	statePtr := func(s model.State) *model.State { return &s }
+	testModels := map[string]db.MockExperimentParams{
+		"project_1_state_active": {
+			ProjectID: &testProjectID,
+			State:     statePtr(model.ActiveState),
+		},
+		"project_1_state_canceled": {
+			ProjectID: &testProjectID,
+			State:     statePtr(model.CanceledState),
+		},
+		"project_1_state_completed": {
+			ProjectID: &testProjectID,
+			State:     statePtr(model.CompletedState),
+		},
+		"project_1_state_error": {
+			ProjectID: &testProjectID,
+			State:     statePtr(model.ErrorState),
+		},
+		"project_1_state_paused": {
+			ProjectID: &testProjectID,
+			State:     statePtr(model.PausedState),
+		},
+		"project_1_state_stoppingCanceled": {
+			ProjectID: &testProjectID,
+			State:     statePtr(model.StoppingCanceledState),
+		},
+		"project_1_state_deleting": {
+			ProjectID: &testProjectID,
+			State:     statePtr(model.DeletingState),
+		},
+		"project_1_state_running": {
+			ProjectID: &testProjectID,
+			State:     statePtr(model.RunningState),
+		},
+		"project_2_state_active": {
+			ProjectID: &testProjectID2,
+			State:     statePtr(model.ActiveState),
+		},
+		"project_2_state_paused": {
+			ProjectID: &testProjectID2,
+			State:     statePtr(model.PausedState),
+		},
 	}
 
-	allExperimentIds := make([]int, len(testModelStates))
-	editableExperiments := map[model.State]*model.Experiment{}
-	for i, state := range testModelStates {
+	allExperimentIds := make([]int, len(testModels))
+	editableExperiments := map[string]*model.Experiment{}
+	i := 0
+	for name, model := range testModels {
 		exp := db.RequireMockExperimentParams(
 			t, db.SingleDB(), testUser,
-			db.MockExperimentParams{
-				State: &state,
-			},
-			testProjectID,
+			model,
+			*model.ProjectID,
 		)
-		editableExperiments[state] = exp
+		editableExperiments[name] = exp
 		allExperimentIds[i] = exp.ID
+		i++
```

Comment:
The most minor of nits: you could get rid of `i` if you allocate `allExperimentIds` with len of zero and capacity of `len(testModels)` and then just `append()`.
```go
allExperimentIds := make([]int, 0, len(testModels))
editableExperiments := map[string]*model.Experiment{}

for name, model := range testModels {
	exp := db.RequireMockExperimentParams(
		t, db.SingleDB(), testUser,
		model,
		*model.ProjectID,
	)
	editableExperiments[name] = exp
	allExperimentIds = append(allExperimentIds, exp.ID)
}
```